src/wp-includes/block-editor.php

```diff
@@ -299,7 +299,7 @@ function get_legacy_widget_block_editor_settings() {
  * }
  */
 function _wp_get_iframed_editor_assets() {
-	global $wp_styles, $wp_scripts, $pagenow;
+	global $wp_styles, $wp_scripts;
```

Comment:
@Mamaduka Remove the global annotation from document.

```
* @global string     $pagenow    The filename of the current screen.
```

Reply:
Thanks @mukeshpanchal27. I'll take care of this when committing.